bears/stylus/StylintBear.py

```diff
@@ -1,6 +1,12 @@
+import json
+
 from coalib.bearlib.abstractions.Linter import linter
 from dependency_management.requirements.NpmRequirement import NpmRequirement
 
+setting_map = {True: 'always',
```

Comment:
ah maybe precede it with an underscore so we "mark it as private". Not that people think they can import this :P